maps/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""
+File for initializing and running flask application.
+"""
+
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from config import Config
+
+# Set up server configuration
+app = Flask(__name__)
+app.config.from_object(Config)
+db = SQLAlchemy(app)
+
+if not app.debug and not app.testing:
+
+    if not os.path.exists('logs'):
+        os.mkdir('logs')
+    handler = RotatingFileHandler('logs/maps.log', maxBytes=10240, backupCount=1)
```

Comment:
Needs some sort of silent mode, I see the app.debug and app.testing if statement, but I'm not sure what it does. I just want to make sure my apache logs aren't spammed.